### ISSUE TITLE
Not fair to call some games clones? Needs discussion.

### ISSUE THREAD

**Issue by Hiradur** [opengaming/osgameclones#406]
I realized I may have been a little bit to quick with adding clones. All I wanted is to spread awareness for some FOSS games. However, Duion, the Uebergame developer told me on IRC he sees Uebergame as an original game, not a clone of ARMA or anything else. He is fine with keeping it in the list for promotion for now though.
However, this made me think. Some of the games I added look pretty similar to the originals (to me) but since I didn't play most of them or didn't play them long enough they may not actually be clones or don't want to be considered clones by their developers/community.

Games in question are:
- Annex: Conquer The World: Does not list CnC as inspiration or anything but the game looks very similar, one community member said that listing it as a clone is probably fine and could make Annex more popular
- Bridge Command (seems to be a very complex simulator, not sure how it compares to the Ship Simulator series)
- Stunt Rally (doesn't look exactly like TrackMania but IMO it definitely has some elements from it)
- Rigs of Rods (BeamNG.drive actually is a commercial project started by the original Rigs of Rods devs after RoR. The focus of the games seems to be slightly different. Rigs of Rods strives to be a hardcore simulator while BNG wants to be a bit more accessible I think (don't hold me on that, I haven't played it and don't want to start a flamewar ;) ). Being a former Rigs of Rods developer I can say that there have always been tensions and flamewars between the two communities but the games look similar in some aspects)
- Spring 1944: Lists both Company of Heroes and Supreme Commander as an inspiration so calling it a clone of CoH may not be right (although I wrote about CoH and SC being inspirations in the info section)
- The Butterfly Effect (this one does look _very_ similar to Crazy Machines and The Incredible Machine so listing it as a clone may be fine although these games are nowhere listed as an inspiration)

What do you think should be done about this? Should I ask the developers of these games if they approve their games being listed as clones? I don't want to do any damage, e.g. that people join the project with wrong expectations. Or should these games rather be moved to the loosely related games category which I proposed in #398? IMO this would be the best option.

**Comment by cxong** [comment]
For obvious clones there's no problem. When there's doubt, I would get a second opinion, from the developers or from players very familiar with the games. Then when there's still doubt, we can look at them on a case-by-case basis.

For uebergame, I bet the *real* objection is the negative connotation of the word "clone". So I think we should avoid using that word altogether, and renaming the section "List of games inspired by" or something like that. "Clones" may be factually accurate but the word has baggage.

**Comment by foobar13373** [comment]
I'd just rename how the categories are called currently:
* "Remake" -> "Engine re-implementation" (only games that actually need or are able to use original game files are put into this category)
* "Clones" => "Inspired by" (any game that can not use original assets, even if it's a 1:1 remake)

**Comment by smcameron** [comment]
You might want to explain what exactly you mean by "clone" and "remake" somewhere on the web page. Also, If you don't already know that "clones" is further down on the web page, you might miss that it is even there, and assume the "remakes" for a particular game are all that exist, since they come first on the page (maybe two columns? Maybe do not divide the list into two separate lists, but classify each game as either a remake or clone, but all in one big list?)

If I look for Artemis Spaceship Bridge Simulator, I see in "remakes" that there is Empty Epsilon.  I might not know to keep scrolling down to find the "clones."

For instance, I see Empty Epsilon classified as a "remake" (whatever that means) of Artemis Spaceship Bridge Simulator, and Space Nerds In Space (my game) classified as a "clone" of Artemis. This seems a bit strange to me. Empty Epsilon is closer to Artemis than Space Nerds In Space is, but "clone" implies (to me) that Space Nerds In Space would be attempting to be identical to Artemis as much as possible (which is absolutely not the case.)

**Comment by fritzy** [comment]
Sometimes games create entire new genres, and it's hard to shake the "clone" moniker.

**Comment by tzizi** [comment]
How about simply "clone" and "similar"? Clones being games that try to remake the original game, and "similar" being games that are similar, possibly inspired by the original game, but not necessarily a clone?

**Comment by smcameron** [comment]
Apparently, "clone" and "remake" are the only allowed options for type.  See:
https://github.com/opengaming/osgameclones/blob/master/schema/games.yaml#L19

So, either add some new types, and use "clone" to mean what it actually means (genetically identical), or else do not mention Space Nerds In Space at all.

**Comment by wonea** [comment]
Yes I agree with inspired addition. Clone and remake are arguably derivatives.

**Comment by tzizi** [comment]
I've been thinking about this, how about something like this?

* Remake: Remakes the game engine, and requires the original files (can be optional, for example, OpenRA optionally provides their own data files).
* Clone: Doesn't remake the game engine, but tries to reproduce the original as faithfully as possible. WarMUX and Pingus could be examples. You're basically getting the same kind of gameplay as you would if you bought the original game.
* Similar: Similar gameplay in some way or another (inspired by, predecessor to, etc.)

I personally think having `inspired`, `similar` and `predecessor` would be a bit redundant. From the provided definition, `inspired` and `similar` seem to be quite similar for the end user.

`predecessor` is seems to only apply to a few very specific games as well, as far as I know, very commercial few games have stemmed from open-source games. Personally I would suggest writing something in the `info` field about it.

**Comment by cxong** [comment]
Ok; going with just `similar` for now. Please reopen if you would like to discuss further.

**Comment by specing** [comment]
Spring: 1944 (short: S44) lists "Think of it like a cross between Company of Heroes and Supreme Commander, as designed by someone who played neither.". I believe this wording was used to quickly describe S44's gameplay to others and not as "we took their gameplay as the basis of our game".

In fact, the Spring: 1944 project started out around 2003 and was already playable as early as 2005 as the "Axis & Allies TA" "mod" for Total Annihilation (1997). This was three years before CoH (2006) was released, and thus it would be more correct to call CoH a clone of S44. Today S44 has a total of 8 playable factions (USA, GBR, SU/RUS, Germany, Japan, Italy, Sweden and Hungary) with aerial and naval warfare. It is undoubtedly a much larger game than CoH (just USA & germany?, land only?).
